src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/mgr-modules/mgr-module-details/mgr-module-details.component.ts

```diff
@@ -19,6 +19,8 @@ export class MgrModuleDetailsComponent implements OnChanges {
     if (this.selection) {
       this.mgrModuleService.getConfig(this.selection.name).subscribe((resp: any) => {
         this.module_config = resp;
+        delete this.module_config.username;
+        delete this.module_config.password;
```

Comment:
Wow, @aaSharma14 , what happened here? A couple of days ago it was a back-end refactoring and now it's just front-end hiding back-end stuff?

Reply:
Back to back-end refactoring.